src/main/resources/assets/securitycraft/lang/fr_fr.json

```diff
@@ -1196,4 +1226,6 @@
 	"securitycraft.subtitle.lock": "Verrouillage du conteneur",
 	"securitycraft.subtitle.get_item": "Objet obtenu !",
 	"securitycraft.subtitle.ping": "Ping du système de sécurité sonique"

```

Comment:
There needs to be a comma at the end of this line for the JSON to be valid. (You can use https://jsonformatter.org to verify validity)